src/sdk-reference/cpp/1/protocol/register-sub/index.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: sdk.html.hbs
+title: registerSub
+description: Used when subscribing to store a listener.
+---
+
+# registerSub
+
+Used when subscribing to store a listener. 
```

Comment:
I am not sure I understand what this does. If I'm correct, then this suggestion should also be applied to this page's description:

`Attaches a notifications listener to an existing subscription.`